Snapper.Json/JsonDiff.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using DiffMatchPatch;
+using Newtonsoft.Json.Linq;
+
+namespace Snapper.Json
+{
+    public class JsonDiff

```

Comment:
This shouldn't be a public class. 
The API for the `Snapper.Json` nuget package shouldn't expose a class called JsonDiff for use. 

I can only think of two options to avoid this. 
1. Make a new library which is used by the two projects.
2. Make this class internal and use the `InternalsVisibleTo` attribute so that the other projects can use it. 

Thoughts?

Reply:
I think I'd go with the second approach in order to not overcomplicate the project structure, if thats fine with you?